### PR TITLE
feat(combo): provide component documentation

### PR DESCRIPTION
**Pull request by SisIvanova**

### CODE REVIEW

File: doc/en/components/grids/combo/overview.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Part name | Description
+---------|------------
+`target` | The encapsulated igc-input that holds the list of selected options.
+`prefix`  | A prefix wrapper that renders content before the target input.
+`suffix` | A suffix wrapper that renders content after the target input.
+`clear-icon` | A clear-icon wrapper that renders content inside the suffix wrapper.
+`toggle-icon` | A toggle-icon wrapper that renders content inside the suffix wrapper.
+`helper-text` | A helper-text wrapper that renders content below the target input.
+`filter-input` | The encapsulated igc-input that holds the filter value.
+`list-wrapper` | A wrapper that holds the filter-input and the `igc-combo-list` component.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/combo/overview.md:122]: replace `igc-combo-list` with `ComboList`

File: doc/en/components/grids/combo/overview.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+> [!Note]
+> When the data source is comprised of a simple type (e.g. `string[]`, `number[]`), **do not** specify a `valueKey` and `displayKey`.
+
+### Selection API
+
+The combobox component exposes API that allows getting and manipulating the current selection state of the control.
+
+Except for selecting combo items from the list options manually, users can select items programmatically. One way to do this is via the combo `select()` method. You can pass an array of items that should be selected or an empty array if you want to select all of the options in the combobox list.
+
+The reversed action is also available using the combo `deselect()` method. In this case, if an empty array is passed, all of the options in the combobox list will be deselected.
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/combo/overview.md:81]: remove "()"

File: doc/en/components/grids/combo/features.md [hunk 3]
```diff
@@ -0,0 +1,175 @@
+---
+title: {Platform} Combo Component – {ProductName}
+_description: {ProductName} Combo Component Features
+_keywords: {ProductName}, UI controls, {Platform} widgets, web widgets, UI widgets, {Platform}, Native {Platform} Components Suite, Native {Platform} Controls, Native {Platform} Components Library, {Platform} Combo Component Features
+mentionedTypes: ['Combo']
```
Comment (HUSSAR-mtrela) [inline doc/en/components/grids/combo/features.md:5]: add 'ComboItem' and 'ComboList' to mentionedTypes

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]
use PascalCase in API links mentioned in **ALL** paragraphs, e.g. 
GroupKey instead of `group-key` 
FilterKey instead of `filterKey`

**Review by simeonoff — APPROVED** [review]

File: doc/en/components/grids/combo/overview.md [hunk 4]
```diff
@@ -0,0 +1,228 @@
+---
+title: {Platform} Combobox Component – {ProductName}
+_description: {Platform} Combo component provides a powerful input, combining features of the basic HTML input, select, filtering and custom drop-down lists. Try it for FREE
+_keywords: {ProductName}, UI controls, {Platform} widgets, web widgets, UI widgets, {Platform}, Native {Platform} Components Suite, Native {Platform} Controls, Native {Platform} Components Library, {Platform} ComboBox component
+mentionedTypes: ['Combo']
+---
+
+# {Platform} ComboBox Overview
+
+{Platform} ComboBox is a lightweight editor that enables users to easily select, filter, and group different predefined options in a provided list. The component also supports options for {Platform} ComboBox Keyboard navigation, templates to customize how the items, header, and footer are displayed, and allow users to write custom values.
```
Comment (SisIvanova) [inline doc/en/components/grids/combo/overview.md:10]: @radomirchev The custom values feature is not yet implemented.

**Comment by ChronosSF** [comment]
@SisIvanova , these topics shouldn't be under grids, should they?

**Comment by ChronosSF** [comment]
@HUSSAR-mtrela , I wanted to test whether or not the API refs generate links properly following your previous review but they all seem to not generate any links. Then I remembered there was some fundamental issue with the api links for WC. Am I remembering correctly and can we merge it now?

TBH without the links the props will make no sense in pascal case so I am a bit worried about the docs.

**Comment by ChronosSF** [comment]
Interestingly digging into this further I noticed that from the following paragraph:

The {ProductName} Combo component supports most of the `Input` properties, such as `Required`, `Disabled`, `Autofocus`, `Invalid`, etc. The component also exposes two methods bound to its validation:

the first two properties don't generate links, the latter two do, but all 4 seem to be present in the api page the ones that do point to.

So there seems to be some issue in the generator...

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**